lib/column/ipv4_test.go

```diff
@@ -0,0 +1,211 @@
+package column
+
+import (
+	"net"
+	"testing"
+)
+
+const invalidIPv4Str = "44.489.38.222"
+
+func TestIPv4_AppendRow_InvalidIP(t *testing.T) {
+	col := IPv4{}
+
+	// appending string
+	err := col.AppendRow(invalidIPv4Str)
+	if err == nil {
+		t.Fatalf("AppendRow didn't IP")
+	}
+	switch e := err.(type) {
+	case *ColumnConverterError:
+		if e.Hint != "invalid IP format" {
+			t.Fatalf("expected error hint %s, got %s", "AppendRow didn't IP", e.Hint)
```

Comment:
can we use `require` instead and avoid indentation? See other tests. e.g. https://github.com/ClickHouse/clickhouse-go/blob/v2/tests/json_test.go#L269

Reply:
Done